kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/IPAddress.java

```diff
@@ -81,6 +83,10 @@ public static IPAddress getByAddress(int addr) throws UnknownHostException {
      * @return
      */
     public static IPAddress parseHostAddress(String hostAddress) throws UnknownHostException {
+        if (!hostAddress.isEmpty() && !InetAddresses.isInetAddress(hostAddress)) {
```

Comment:
Why is this needed?

Reply:
This may fail in certain host configurations.